frontend/talentsearch/src/js/components/search/deprecated/SearchForm.tsx

```diff
@@ -258,8 +262,20 @@ export const SearchForm = React.forwardRef<SearchFormRef, SearchFormProps>(
                 description:
                   "Placeholder for classification filter in search form.",
               })}
-              name="classifications"
-              options={classificationOptions}
+              name="classification"
+              options={[
+                {
+                  value: "",
+                  disabled: true,
+                  label: intl.formatMessage({
+                    defaultMessage: "Select one or more classification(s)",
```

Comment:
This placeholder needs to change, along with its translation, and L260.

Reply:
Fixed with [update classification label](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3872/commits/0f879693fb970f01ca8af58067c1c462392f425a). New translation was run by Gray.